### ISSUE TITLE
[Bug] The timeago is displayed incorrectly

### ISSUE THREAD

**Issue by Xiaokang2022** [timvink/mkdocs-git-revision-date-localized-plugin#142]
mkdocs-git-revision-date-localized-plugin: `1.2.7`

I noticed that version `1.2.7` fixed a [related bug](https://github.com/timvink/mkdocs-git-revision-date-localized-plugin/pull/141), but it still has a bug.

Here's a part of my `mkdocs.yml`:

```yaml
theme:
  name: material
  language: zh

plugins:
  - git-revision-date-localized:
      type: timeago
```

According to https://timvink.github.io/mkdocs-git-revision-date-localized-plugin/howto/specify-locale/, the timeago should shows as `zh`, but it still shows as `en`.

I have to specify the locale explicitly. This is not as described in the documentation.

**Comment by Xiaokang2022** [comment]
Hopefully this bug can be fixed soon.

**Comment by Xiaokang2022** [comment]
I forked the repository and created a PR to try and fix the problem: #143

**Comment by timvink** [comment]
I see. 

so mkdocs specifies a `theme: locale`  parameter (https://www.mkdocs.org/user-guide/localizing-your-theme/) while mkdocs-material uses `theme: language` (https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language)

Tested the PR and it works.

Probably under the hood mkdocs has started defining a `locale` default, and this it was not none. Thanks for fixing!
